src/sentry/static/sentry/app/utils/discover/genericDiscoverQuery.tsx

```diff
@@ -153,6 +159,9 @@ class GenericDiscoverQuery<T, P> extends React.Component<Props<T, P>, State<T>>
     if (limit) {
       apiPayload.per_page = limit;
     }
+    if (noPagination) {
+      apiPayload.no_pagination = noPagination;
```

Comment:
Most of the API parameters are camelBacked as well. So `noPagination` in this case.